docs/Developer-Docs/Pytest-CI-Integration.rst

```diff
@@ -0,0 +1,27 @@
+PyTest CI Integration
```

Comment:
Maybe "Manager Tests" / "Manager development" in the same vein as the other driver/compiler version of this.

Reply:
Done